### ISSUE TITLE
twisted.web.client.Agent's docstring incorrectly claims it doesn't verify certificates

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7750]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#7750|
|Type|defect|
|Created|2015-01-16 06:34:33Z|
|Branch|https://github.com/twisted/twisted/tree/fix-agent-docstring-7750|

`L{Agent} is a very basic HTTP client.  It supports I{HTTP} and I{HTTPS} scheme URIs (but performs no certificate checking by default).` is fortunately wrong since 14.0.

We fixed this so we shouldn’t confuse/discourage people from using it. :)

<details><summary>Searchable metadata</summary>

```
trac-id__7750 7750
type__defect defect
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__branches_fix_agent_docstring_7750 branches/fix-agent-docstring-7750
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1421390073193148 1421390073193148
changetime__1421484607789760 1421484607789760
version__None None
owner__hynek hynek
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

(In [43670]) Branching to fix-agent-docstring-7750.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @hynek|
|-|-|

Thanks hynek.

Looks good to me, buildbot is happy, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_5"></a>|@hynek set status to `closed`|
|-|-|

(In [43674]) Merge fix-agent-docstring-7750

Author: hynek
Reviewer: glyph
Fixes: [#7750](https://github.com/twisted/twisted/issues/7750)
